conda/cli/conda_argparse.py

```diff
@@ -123,9 +124,21 @@ def do_call(args, parser):
 
     module = import_module(relative_mod, __name__.rsplit(".", 1)[0])
 
+    command = relative_mod.replace(".main_", "")
+    _run_pre_command_hooks(command, args)
+
     return getattr(module, func_name)(args, parser)
 
 
+def _run_pre_command_hooks(command: str, args) -> None:
+    """
+    Helper function used to gather applicable pre_command hook functions
+    and then run them.
+    """
+    actions = context.plugin_manager.yield_pre_command_hook_actions(command)
+    tuple(action(command, args) for action in actions)
```

Comment:
I prefer expanded for-loops instead of list comprehensions if we arent returning, storing, or otherwise using the results, I find it to be less error prone (e.g., dropping the `tuple` here would make this a generator that never executes)

```suggestion
    for action in actions:
        action(command, args)
```

Reply:
Gave that feedback in my previous review as well 🤣

Reply:
What can I say. I'm a sucker for one liners 😏... Will change it 👌 .